.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 language: python
 cache: pip
 python:
- - "3.5"
```

Comment:
I figured 3.5 wasn't necessary to build, might as well just use 3.6 but maybe you have a reason for both 🤷‍♂